From 2c1d783d5def98b5f6f83022882747d542a83b82 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 8 Oct 2014 14:04:49 -0400
Subject: [redhat] kabi: Adding symbol hrtimer_cancel (kernel/hrtimer.c)

Message-id: <69fd81db72cb10d84c4bf35a47db9ed44b571c8c.1412771859.git.poros@redhat.com>
Patchwork-id: 97182
O-Subject: [RHEL7.1 PATCH 29/67] kabi: Adding symbol hrtimer_cancel (kernel/hrtimer.c)
Bugzilla: 1148054
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

Bugzilla: 1148054
    https://bugzilla.redhat.com/show_bug.cgi?id=1148054

    Description:
    ============
    Cancel timer events that require HR timer.
    Symbol used in extint_timer.ko

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/hrtimer_cancel
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/hrtimer_cancel
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/hrtimer_cancel

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/hrtimer_cancel b/redhat/kabi/kabi-module/kabi_ppc64/hrtimer_cancel
new file mode 100644
index 0000000..839c728
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/hrtimer_cancel
@@ -0,0 +1,2 @@
+#1-
+0xc1647dcc hrtimer_cancel vmlinux EXPORT_SYMBOL_GPL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/hrtimer_cancel b/redhat/kabi/kabi-module/kabi_s390x/hrtimer_cancel
new file mode 100644
index 0000000..0bb37b2
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/hrtimer_cancel
@@ -0,0 +1,2 @@
+#1-
+0x4cdca450 hrtimer_cancel vmlinux EXPORT_SYMBOL_GPL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/hrtimer_cancel b/redhat/kabi/kabi-module/kabi_x86_64/hrtimer_cancel
new file mode 100644
index 0000000..1272ce3
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/hrtimer_cancel
@@ -0,0 +1,2 @@
+#1-
+0x094313d7 hrtimer_cancel vmlinux EXPORT_SYMBOL_GPL
-- 
1.7.1